build.cake

```diff
@@ -19,17 +24,23 @@ Task("Clean")
     });
 
 // Copy everything to build so we make no changes in the actual files.
+Task("CopyGenerators")
+    .Does(() => {
+        CopyDirectory($"{generatorsSourceDir}/", $"{generatorsBuildDir}");
+    });
+
```

Comment:
We don't perform any file replacements in the generators project like we do in the exercises (replacing the example with the stub). There shouldn't be any need to copy the generators directory.